src/cmd/cmd.zig

```diff
@@ -1123,6 +1124,8 @@ const AppBase = struct {
         errdefer entrypoints.deinit();
 
         const ip_echo_data = try getMyDataFromIpEcho(logger, entrypoints.items);
+        // Try to extract the port from the given host. If no port is found, fall back to the configured default port.
+        const my_port = (config.current.gossip.getPortFromHost() orelse config.current.gossip.port) catch config.current.gossip.port;
```

Comment:
Another option I considered was to error out if `--gossip-host` is set (which must have a port portion) and if `--gossip-port`.

Yet another option is to drop the requirement that `--gossip-host` require the port portion.

Out of all these, I felt this fallback was the least intrusive approach.